frontend/src/components/dropdowns/InfoSessionDropdownField/index.js

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+
+import GenericDropdownField from '../base';
+
+import INFO_SESSIONS from '@Constants/infoSessions';
+
+
+const InfoSessionDropdownField = props => {
```

Comment:
I've gone over this before but we should all be making our files Typescript files if possible i.e. make it a Typescript file unless we HAVE TO make it a js file like a config file for a package we use.